zerver/lib/hotspots.py

```diff
@@ -75,26 +79,46 @@ def get_next_hotspots(user: UserProfile) -> List[Dict[str, object]]:
     if not settings.TUTORIAL_ENABLED:
         return []
 
-    if user.tutorial_status == UserProfile.TUTORIAL_FINISHED:
-        return []
-
     seen_hotspots = frozenset(
         UserHotspot.objects.filter(user=user).values_list("hotspot", flat=True)
     )
-    for hotspot in INTRO_HOTSPOTS.keys():
-        if hotspot not in seen_hotspots:
-            return [
-                {
-                    "name": hotspot,
-                    "title": str(INTRO_HOTSPOTS[hotspot]["title"]),
-                    "description": str(INTRO_HOTSPOTS[hotspot]["description"]),
-                    "delay": 0.5,
-                }
-            ]
+
+    hotspots = []
+
+    for name, base_hotspot in NON_INTRO_HOTSPOTS.items():
+        if name in seen_hotspots:
+            continue
+
+        hotspot = {
+            **base_hotspot,
+            "name": name,
+            "title": str(base_hotspot["title"]),
+            "description": str(base_hotspot["description"]),
+            "is_non_intro_hotspot": True,
```

Comment:
Using `is_non_intro_hotspot` solved the mypy error.

Reply:
I am thinking maybe we should change `is_non_intro_hotspot` -> `show_icon`. 
As `is_non_intro_hotspot` is only used to do that in the front end. 

(I'll update it tomorrow morning!)

Reply:
I think calling it e.g. `has_trigger: True` or something else declarative about the nature of this specific hotspot would probably be more appropriate.